io_scene_nif/__init__.py

```diff
@@ -48,26 +48,18 @@
 _dependencies_path = os.path.join(os.path.dirname(__file__), "dependencies")
 if _dependencies_path not in sys.path:
     sys.path.append(_dependencies_path)
-    print(sys.path)
 del _dependencies_path
 
 import io_scene_nif
 from io_scene_nif import properties, operators, ui
 
-
-try:
-    from io_scene_nif.utility import nif_debug
-    nif_debug.start_debug()
-except:
-    print("Failed to load debug module")
-
 # Blender addon info.
 bl_info = {
     "name": "NetImmerse/Gamebryo nif format",
     "description": "Import and export files in the NetImmerse/Gamebryo nif format (.nif)",
     "author": "NifTools Team",
     "version": (2, 6, 0),  # can't read from VERSION, blender wants it hardcoded
-    "blender": (2, 7, 7),
+    "blender": (2, 79, 0),
```

Comment:
I do believe this resolves #258

Reply:
We'd need to do some testing on earlier verison tbh.
#258 changed focus though to automate version compatibility for the plugin.
Though I wouldn't bother with any 2.7x version now with 2.8x on the horizon with all the BICs.